### ISSUE TITLE
[BUG]: VstsAzureHelpers_ --> InitializeAzModuleFunctions.ps1 outputs object type and not content when using write-host

### ISSUE THREAD

**Issue by mortenlerudjordet** [microsoft/azure-pipelines-tasks#19696]
### New issue checklist

- [X] I searched for [existing GitHub issues](https://github.com/microsoft/azure-pipelines-tasks/issues)
- [X] I read [pipeline troubleshooting guide](https://docs.microsoft.com/vsts/build-release/actions/troubleshooting)
- [X] I checked [how to collect logs](https://learn.microsoft.com/azure/devops/pipelines/troubleshooting/review-logs?view=azure-devops)

### Task name

common/VstsAzureHelpers_

### Task version

latest

### Issue Description

In InitializeAzModuleFunctions.ps1 following code needs to be changed.

From:
`Write-Host "##[command]$command $args"`
To:
`Write-Host "##[command]$command $($args | Out-String)"`

$args is a hashtable and using it inside quotation marks on write-host will only output object type to terminal and not the contents of it. By piping to out-string will change this behavior. 

### Environment type (Please select at least one enviroment where you face this issue)

- [X] Self-Hosted
- [X] Microsoft Hosted
- [ ] VMSS Pool
- [ ] Container

### Azure DevOps Server type

dev.azure.com (formerly visualstudio.com)

### Operation system

All

### Relevant log output

```shell
From AzurePowershellV5 output using the helper function:


Connect-AzAccount System.Collections.Hashtable
```
```[tasklist]
### Tasks
```

**Comment by mortenlerudjordet** [comment]
Added change in PR: #19712

**Comment by mortenlerudjordet** [comment]
Merged with PR: #19762
